src/shared/firewall-util.c

```diff
@@ -76,8 +76,11 @@ static int entry_fill_basics(
         }
 
         if (out_interface) {
+                unsigned l = strlen(out_interface);
```

Comment:
this doesn't look right. strlen() returns `size_t`, not `unsigned` which is 64bit vs 32bit on x86-64...